CompilerSource/backend/GameData.cpp

```diff
@@ -306,7 +306,7 @@ ObjectData::ObjectData(const ::GmObject &object, const ESLookup &lookup):
 
 RoomData::RoomData(const buffers::resources::Room &q, const std::string& name):
   BaseProtoWrapper(q), name(name) {}
-RoomData::RoomData(const ::Room &room, const ESLookup &lookup):
+RoomData::RoomData(const deprecated::JavaStruct::Room &room, const ESLookup &lookup):
```

Comment:
Again, could use a using here. I mean this is a source and not a header so doing it here wouldn't be as bad.

Reply:
I'm happy to leave it explicit for the remainder of its life.